lib/msal-browser/src/version.json

```diff
@@ -0,0 +1 @@
+{"name":"@azure/msal-browser","version":"2.9.0"}
```

Comment:
Is there a reason we can't just take the version from the package.json?

Reply:
We shouldn't be emitting things in the build that live outside the root directory and some tools like tsdx require that the root directory is set to "./src". Though this isn't causing problems for browser today it is for other libs so the change here is mainly to keep all our libraries aligned